modules/machineset-yaml-rhv.adoc

```diff
@@ -93,39 +95,45 @@ The `instance_type_id` field is deprecated and will be removed in a future relea
 ====
 +
 If you include this parameter, you do not need to specify the hardware parameters of the VM including CPU and memory because this parameter overrides all hardware parameters.
+<9> Setting this option to `false` enables preallocation of disks. The default is `true`. Setting `sparse` to `true` with `format` set to `raw` is not available for block storage domains. The 'raw' format writes the entire virtual disk to the underlying physical disk.
+<10> Can be set to `cow` or `raw`. The default is `cow`. The 'cow' format is optimized for virtual machines.
++
+[NOTE]
+====
+Preallocating disks on file storage domains writes zeroes to the file. This might not actually preallocate disks depending on the underlying storage.
+====
+<11> Optional: The CPU field contains the CPU's configuration, including sockets, cores, and threads.
```

Comment:
You could change "the CPU's configuration" to "the configuration of the CPU", to avoid using a possessive. My understanding is that this helps with localisation. My apologies if I missed this in my initial review.

Reply:
I completely agree - I try to avoid any use of apostrophes and quotes in our documentation.